.github/workflows/pr_label_check.yml

```diff
@@ -4,6 +4,9 @@ name: PR label check
 on:
   pull_request:
     types:
+      - opened
+      - synchronize
+      - reopened
```

Comment:
There is no need to add these events.

Reply:
These three have been removed from the list of options.